src/opendr/perception/compressive_learning/multilinear_compressive_learning/multilinear_compressive_learner.py

```diff
@@ -405,7 +405,7 @@ def infer_from_compressed_measurement(self, measurement):
         self.model.to(torch.device(self.device))
         self.model.eval()
 
-        measurement = measurement.numpy()
+        measurement = measurement.convert("channels_last")
```

Comment:
Is this missing an explicit conversion to RGB similar to the lines above? Alternatively, remove the parameter above as it is the default anyway.

Reply:
This is a bit strange case. We no longer have RGB channels, since this is a compressed form of the image (indeed, in testing 2 channels are used). By using the default parameters we do not perform any RGB2BGR conversion, but we indeed get a different channel ordering.

Reply:
Got it. Thanks for the explanation.